src/shared/libcrypt-util.c

```diff
@@ -21,6 +21,7 @@
 #include "errno-util.h"
 #include "libcrypt-util.h"
 #include "log.h"
+#include "random-util.h"
```

Comment:
Why does it work as-is right now if this is needed?

Reply:
As all our CIs test the branch that `HAVE_CRYPT_GENSALT_RA` is true. crypto_random_bytes() defined in random-util.h is used only when the other branch.

Reply:
Please note this in the commit message, otherwise it's unclear

Reply:
Done.